tsdb/engine/tsm1/pools.go

```diff
@@ -145,3 +137,36 @@ func putValue(buf []Value) {
 		}
 	}
 }
+
+type BytesPool struct {
```

Comment:
Is this duplicated from `pool.Bytes` for a reason? Or was this just the original place your wrote it? :-)